.github/workflows/abq-release.yml

```diff
@@ -35,6 +35,14 @@ jobs:
           registry-url: https://registry.npmjs.org
           scope: '@rwx-research'
           cache: yarn
+          always-auth: true
+      - name: Setup .yarnrc.yml
+        run: |
+          yarn config set npmScopes.rwx-research.npmRegistryServer "https://registry.yarnpkg.com"
```

Comment:
do we want to set the npm registry here?

Reply:
I pulled that from the setup-node docs, and as far as I can tell it's just a caching front for npmjs. I've removed it from this PR.